driver.md

```diff
@@ -84,12 +84,36 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
            <sigR>       TR   </sigR>
            <sigS>       TS   </sigS>
            <data>       DATA </data>
+           <txType>     0    </txType>
+           ...
+         </message>
+
+    rule <k> startTx => loadTx(#sender(TYPE, CID, TN, TP, TG, TT, TV, #unparseByteStack(DATA), TA, TW, TR, TS)) ... </k>
+         <chainID> CID </chainID>
+         <txPending> ListItem(TXID:Int) ... </txPending>
+         <message>
+           <msgID>      TXID </msgID>
+           <txNonce>    TN   </txNonce>
+           <txGasPrice> TP   </txGasPrice>
+           <txGasLimit> TG   </txGasLimit>
+           <to>         TT   </to>
+           <value>      TV   </value>
+           <sigV>       TW   </sigV>
+           <sigR>       TR   </sigR>
+           <sigS>       TS   </sigS>
+           <data>       DATA </data>
+           <txType>     TYPE </txType>
+           <txChainID>  CID  </txChainID>
+           <txAccess>   TA   </txAccess>
+           ...
          </message>
+    requires TYPE ==Int 1
 
     syntax EthereumCommand ::= loadTx ( Account )
  // ---------------------------------------------
```

Comment:
I think I would prefer if this set the `<touchedAccounts>` cell to `.Set`, and then we used `#touchAccounts` for the three addresses to populate it with. You can add a 3-argument `#touchAccounts _ _ _` to do this.